### PR TITLE
remove duplicated import

### PR DESCRIPTION
**Pull request by stewart-yu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
remove duplicated import

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/sample-controller/controller.go [hunk 1]
```diff
@@ -150,7 +149,7 @@ func NewController(
 // is closed, at which point it will shutdown the workqueue and wait for
 // workers to finish processing their current work items.
 func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
-	defer runtime.HandleCrash()
+	defer utilruntime.HandleCrash()
```
Comment (nikhita) [inline staging/src/k8s.io/sample-controller/controller.go]: Wish Go would catch these automatically...

**Review by 0xmichalis — APPROVED** [review]
/lgtm

**Comment by nikhita** [comment]
/uncc @kargakis 

/priority important-longterm
/lgtm

/assign @sttts 
for approval

**Comment by nikhita** [comment]
Oops, looks like a race of `lgtm`s. :smile:

**Pull request merged**